### PR TITLE
sip: add tests for headers sticky buffers v2

### PR DESCRIPTION
**Pull request by glongo**
Ticket #6374

## Ticket

If your pull request is related to a Suricata ticket, please provide
the full URL to the ticket here so this pull request can monitor
changes to the ticket status:

Redmine ticket:
https://redmine.openinfosecfoundation.org/issues/6374

### CODE REVIEW

**Comment by glongo** [comment]
Suricata PR: https://github.com/OISF/suricata/pull/10907

**Comment by glongo** [comment]
Suricata PR: https://github.com/OISF/suricata/pull/11004

File: tests/sip-to/test.rules [hunk 1]
```diff
@@ -0,0 +1,2 @@
+alert sip any any -> any any (flow:to_server; sip.to; content:"sip:"; sid:1;)
```
Comment (catenacyber) [inline tests/sip-to/test.rules:1]: Could we have a more precise content ? This looks a bit too generic...

File: tests/sip-content-length/test.rules [hunk 2]
```diff
@@ -0,0 +1,2 @@
+alert sip any any -> any any (flow:to_server; sip.content_length; content:"270"; sid:1;)
```
Comment (catenacyber) [inline tests/sip-content-length/test.rules:1]: Should we try to convert it to an integer ?

**Pull request closed without merging**